git.lua

```diff
@@ -258,15 +264,29 @@ local stashes = function(token)  -- luacheck: no unused args
     -- generate matches and match filter table
     local ret = {}
     local ret_filter = {}
+    local clink_version_encoded = clink.version_encoded or 0
     for i,v in ipairs(stash_times) do
-        table.insert(ret, "stash@{"..(i-1).."}")
-        table.insert(ret_filter, "stash@{"..(i-1).."}    "..stashes[v])
+        local match = "stash@{"..(i-1).."}"
+        table.insert(ret, match)
+        if clink_version_encoded >= 10010012 then
+            table.insert(ret_filter, { match=match, type="word", description=stashes[v] })
```

Comment:
Did clink interface to add completions change in this version? If so why does this apply only to `git stash` completions? How does this affect other commands completions?

Reply:
Yes, big changes.  Backward compatibility exists, so the existing stuff all continues to work.  The changes here make use of new functionality when available.